signing-providers/index.html

```diff
@@ -176,6 +176,8 @@ <h3>Hardware Wallet</h3>
         </div>
     </div>
 
+    <iframe id="webviewChild" src="http://localhost:63342/mx-sdk-js-examples/signing-providers/webviewChild.html" style="width: 100%; height: 200px;"></iframe>
```

Comment:
`http://localhost:63342` - this might cause some issues. Can we use a relative URL?

Reply:
good point. fixed